invocations/hcppipelines-4.0.0-1_invocation.json

```diff
@@ -0,0 +1,7 @@
+{
+	"bids_dir": "/bids",
+	"output_dir": "/output",
+	"analysis_level": "participant",
+	"license_key": "/license/license.txt",
+	"n_cpus":4
```

Comment:
add stages key here

only PostFreesurfer?
or all of PreFreeSurfer, FreeSurfer, PostFreeSurfer?

Reply:
doesn't it run all stages by default?

Reply:
yes but it'll also run fMRIVolume , fMRISurface and DiffusionPreprocessing, which we don't want